decidim-accountability/app/commands/decidim/accountability/result_command_helper.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module ResultCommandHelper
```

Comment:
I don't like using the Helper suffix for something that isn't a Rails Helper (i.e. Action View Helper). Maybe just leaving it as ResultCommand is enough? As we usually don't add the Command suffix to the name of the commad classes, I think that the intention is clear enough.

Another alternative would be to call it "HasResultCommand" or something like that.

Reply:
done in ce0b3cf444